### DISCUSSION TITLE
pyart.util.columnsect

### DISCUSSION THREAD

**Discussion by kpittman6** [ARM-DOE/pyart#1350] [Q&A] [answered]
Hello everyone,

I'm a bit of a PyART novice, and am looking to identify a value for a field from a Level II NEXRAD file (reflectivity, velocity, etc..) at a user input lat/lon point. I came across the pyart.util.columnsect function in this example (https://arm-doe.github.io/pyart/examples/retrieve/plot_column_subset.html), and I think this would give me exactly what I'm looking for. However, I get a AttributeError: module 'pyart.util' has no attribute 'columnsect' when I try to run the example code, and I don't see this function anywhere in the pyart.util library (https://arm-doe.github.io/pyart/API/generated/pyart.util.html). Did this get removed or renamed?

Thank you!

**Comment by mgrover1** [comment, marked as answer]
@kpittman6 - do you know which version of Py-ART you are running?

**Reply by kpittman6** [reply]
That was the problem, updated version and it works. Sorry about that - should have been the first thing I checked.

**Reply by mgrover1** [reply]
Great - no worries! I am glad things are working for you. Let me know if you run into any other issues. We appreciate the feedback!

**Reply by kpittman6** [reply]
Thanks Max!

The only other issue I'm having is that when I read in Level II .gz files that are older than ~ mid 2008, the radar's physical lat/lon location is set at 0°, 0°. I'm guessing this had to do with the legacy to super res switch on all the NEXRAD data that happened at that time, since all the files in my dataset newer than mid 2008 (which have _V03, _V04, or _V06 in the filename) work fine. Is there an easy way to add the coordinates of the radar itself so that this plots/pulls the column in the correct location? I tried updating the grid_origin argument in the pyart.map.grid_from_radars function, but that doesn't do it.

**Reply by mgrover1** [reply]
Check out this discussion https://github.com/ARM-DOE/pyart/discussions/1273

**Reply by kpittman6** [reply]
Great, thank you.
